psl/parser-database/src/walkers/enum.rs

```diff
@@ -54,7 +54,18 @@ impl<'db> EnumWalker<'db> {
 
     /// What kind of newlines the enum uses.
     pub fn newline(self) -> NewlineType {
-        NewlineType::Unix
+        let value = match self.ast_enum().values.last() {
```

Comment:
Can we pull that into a helper? From the point we got `span` we do idientical thing in all cases.